public/js/scripts.js

```diff
@@ -1,63 +1,44 @@
 $(function() {
 
-  // Variable used to track if there are any outstanding actions that webdriver should wait for during
-  // the acceptance test runs
-  window.csrActive = 0
```

Comment:
@lukastymo Is this really not needed anymore? It was put in for the acceptance tests